canary/src/main/java/io/littlehorse/canary/aggregator/serdes/MetricAverageDeserializer.java

```diff
@@ -0,0 +1,24 @@
+package io.littlehorse.canary.aggregator.serdes;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.littlehorse.canary.aggregator.internal.ProtobufDeserializationException;
+import io.littlehorse.canary.proto.MetricAverage;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.common.serialization.Deserializer;
+
+@Slf4j
+public class MetricAverageDeserializer implements Deserializer<MetricAverage> {
```

Comment:
Interesting to note that you decided not to use `LHSerializable` here. Seems you're using the proto object just to store data, which is actually what it is designed to do.

If I were to re-write the LH Server, I would probably use your pattern.